internal/pkg/cli/prompt/nop/nop.go

```diff
@@ -27,17 +27,25 @@ func (p *Prompt) Ask(q *prompt.Question) (result string, ok bool) {
 }
 
 func (p *Prompt) Select(s *prompt.Select) (value string, ok bool) {
-	return s.Default, true
+	return s.Default, s.UseDefault
 }
 
 func (p *Prompt) SelectIndex(s *prompt.SelectIndex) (index int, ok bool) {
-	return s.Default, true
+	return s.Default, s.UseDefault
```

Comment:
To iste aj tu, ... tu je `default` (index) `int`, by default `0` ... a tu to je este vacsi problem, ... lebo nie je jasne, ci je default hodnota - prva hodnota - `index 0 `, alebo ze nebola zadana.